File: API/API/Models/Flight.cs
namespace API.Models
{
    public class Flight
    {
        public int id { get; set; }
        public int boarding_gate { get; set; }
        public int price { get; set; }
        public string status { get; set; }
        public Route route { get; set; }
        public Airplane airplane { get; set; }
        public Promotion promotions { get; set; }



    }
}

File: TECAir_App_Web/src/app/models/flight.ts
export interface Flight {
  id: number;
  boarding_gate: number;
  price: number;
  status: boolean;
  route_code:number;
  airplane_plate: string;
  origin: string;
  destination: string;
  year: number;
  month: number;
  day: number;
  hours: number;
  minutes: string;
  promotion_code: number;
  flight_id: number;
  final_price:number;
  duration:string;
}
